docker/android-slave-s2i/test/run

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
```

Comment:
Is that from base-s2i image? Can we use image instead of copying file here? 
I'm asking because it would be easier to get that from upstream